documentation/practices.md

```diff
@@ -80,5 +80,37 @@ Configuration properties for content URLs are:
 
 For any other CMS, you must update the configurations to reflect the public URLs of the files being retrieved by the particular content provider.
 
+## Indi Brand Loyalty Example
+
+The React PWA Reference Storefront is pre-configured for integration with Indi, with a custom component created to interact with the various widgets Indi may display. For more information on Indi, visit their official [website](https://indi.com/).
+
+The Indi component available is a stand-alone component created solely to fetch the `indi-embed` libraries, then display the corresponding elements that were passed into the component as a structured list.
+
+The required parameters of this component are as follows:
+* `render`: A list of the elements to display from Indi. Example: `render={['carousel', 'brand', 'product']}`. Note the correct configurations must be provided for the element to render correctly.
```

Comment:
I'm just looking at the comments, but one question I have as someone new (and maybe if I saw the whole page, this would make sense) but in the note, you mention correct configurations are required, is there an example of some configurations?  If I were to do it, I would want a guide.

Reply:
Example provided in `ep.config.json`